package.json

```diff
@@ -141,8 +141,8 @@
     "ovh-manager-webfont": "^1.0.1",
     "ovh-ng-input-password": "^1.2.5",
     "ovh-ngstrap": "^4.0.2",
-    "ovh-ui-angular": "^2.18.0",
-    "ovh-ui-kit": "^2.20.0",
+    "ovh-ui-angular": "^2.21.2",
```

Comment:
According to your comment (https://github.com/ovh-ux/ovh-manager-telecom/pull/981/files#diff-624543a53375b956c14494ef14c33255R40) does this upgrade is needed?

Reply:
Yes it is needed, since the upgrade is related to focus select-picker on initialisation (if ng-model equals value of the component)